### PR TITLE
conformance: test resolver with query about unsigned zone

### PR DESCRIPTION
**Pull request by japaric**

### CODE REVIEW

File: conformance/packages/conformance-tests/src/resolver/dnssec/scenarios/insecure.rs [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+// in this DNS network all zones except one are signed. and importantly, the referral to the
+// unsigned zone (the NS+A records in the parent zone) is also signed
+//
+// a validating resolver should not respond with SERVFAIL to queries about the unsigned zone because
+// the security status of the whole zone is "Insecure", not "Bogus"
+#[test]
+#[ignore]
+fn unsigned_zone() -> Result<()> {
+    let expected_ipv4_addr = Ipv4Addr::new(1, 2, 3, 4);
+    let needle_fqdn = FQDN("example.unsigned.com.")?;
```
Comment (bluejekyll) [inline conformance/packages/conformance-tests/src/resolver/dnssec/scenarios/insecure.rs:18]: This should use the testing TLD from #2359, right?

**Review by bluejekyll — CHANGES_REQUESTED** [review]

File: conformance/packages/conformance-tests/src/resolver/dnssec/scenarios/insecure.rs (hunk 1 above)
Comment (japaric) [reply conformance/packages/conformance-tests/src/resolver/dnssec/scenarios/insecure.rs:18]: once that PR is merged, this domain should be updated, yes.

**Comment by japaric** [comment]
adapted to the domain changes from PR #2359

**Review by listochkin — APPROVED** [review]

**Comment by japaric** [comment]
@bluejekyll I think this can't be merged until you update your review :-)

**Comment by japaric** [comment]
renamed the `com_ns` variables to `tld_ns` as we've changed the TLD from `com.` to `testing.`

**Review by bluejekyll — APPROVED** [review]
looks good. Thanks!

**Pull request merged**